project-generator/src/test/resources/com/redhat/ipaas/project/converter/test-pull-push-funktion.yml

```diff
@@ -4,9 +4,9 @@ flows:
   - kind: endpoint
     uri: periodic-timer:every?period=5000
   - kind: endpoint
-    uri: http-get:http://localhost:8080/hello
+    uri: http-get:?httpUri=http://localhost:8080/hello
   - kind: log
     message: Hello World! ${body}
     loggingLevel: INFO
   - kind: endpoint
-    uri: http-post:http://localhost:8080/bye
+    uri: http-post:?httpUri=http://localhost:8080/bye
```

Comment:
Guess this fix is required here, too. just asking, not that something slips through accidentally :)